### PR TITLE
Adds recipe for ClustImpute package from CRAN

### PR DESCRIPTION
**Pull request by mfansler**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

@conda-forge/help-r 

This recipe was generated using [the conda_r_skeleton_helper](https://github.com/bgruening/conda_r_skeleton_helper) script.

### CODE REVIEW

**Comment by mfansler** [comment]
@conda-forge/help-r

Appears two dependencies, `r-copula` and `r-clusterr`, do not have builds available for `r-base[version'>=4.0,<4.1.0a0']` for **win-64** platform. Please advise on how to proceed. The Conda Forge feedstocks for `r-copula` and `r-clusterr` appear to have disable **win-64** platform altogether.

File: recipes/r-clustimpute/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    - r-rlang
+
+test:
+  commands:
+    - $R -e "library('ClustImpute')"           # [not win]
+    - "\"%R%\" -e \"library('ClustImpute')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=ClustImpute
+  license: GPL-3
```
Comment (dbast) [inline recipes/r-clustimpute/meta.yaml:48]:
```suggestion
  license: GPL-3. 0-only
```

**Comment by dbast** [comment]
One minor comment, see inline. The win failure can be ignored, as the package is noarch and will later after feedstock conversion be built on linux only. To be able to install it on windows, the mentioned dependencies have to be made available for windows.

**Comment by cbrueffer** [comment]
Looks good now, thanks!

**Pull request merged**